src/task/JoinAndReceiveIndexTask.ts

```diff
@@ -75,7 +80,21 @@ export default class JoinAndReceiveIndexTask extends BaseTask {
             }
 
             context.serverSupportsCompression = joinAckFrame?.wantsCompressedSdp;
-
+            if (
+              joinAckFrame?.serverSideNetworkAdaptionOverride !== undefined &&
+              context.videoDownlinkBandwidthPolicy.overrideServerSideNetworkAdaption !== undefined
+            ) {
+              const serverSideNetworkAdaptionOverride: SdkServerSideNetworkAdaption =
+                joinAckFrame?.serverSideNetworkAdaptionOverride;
```

Comment:
nit: No need for ? right since it should be covered in the if statement.

Reply:
Fixed! would have been impossible to get to 100% coverage if left as is :P